solutions/usaco-573.mdx

```diff
@@ -11,6 +11,115 @@ author: Qi Wang
 
 <LanguageSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int number_of_wins(vector<int> &bessie, vector<int> &elsie, bool smaller_card_wins)
+{
+	/*
+	 * if person with larger value cards wins
```

Comment:
add this to the explanation part outside the code instead of here

Reply:
donee